inlong-sort/sort-formats/format-rowdata/format-inlongmsg-rowdata-base/src/main/java/org/apache/inlong/sort/formats/inlongmsg/AbstractInLongMsgFormatDeserializer.java

```diff
@@ -81,6 +88,21 @@ public List<InLongMsgWrap> preParse(byte[] bytes) throws Exception {
         final List<InLongMsgWrap> result = new ArrayList<>();
 
         InLongMsg inLongMsg = InLongMsg.parseFrom(bytes);
+        if (inLongMsg == null) {
+            failureHandler.onParsingMsgFailure(bytes, new IOException(
+                    String.format("Could not parse InLongMsg from bytes. bytes={}.",
+                            StringUtils.join(bytes))));
```

Comment:
Is it to much content to print the bytes?

Reply:
This is the callback processing, no printing

Reply:
![image](https://github.com/user-attachments/assets/a548da81-1918-48d6-b3a8-28df561f0463)
It is bad operation.